atc/lidar/scanner_test.go

```diff
@@ -220,13 +220,8 @@ var _ = Describe("Scanner", func() {
 
 						It("sets the check error", func() {
 							Expect(fakeResourceType.SetCheckSetupErrorCallCount()).To(Equal(1))
-							Expect(fakeResource.SetCheckSetupErrorCallCount()).To(Equal(1))
-							err := fakeResource.SetCheckSetupErrorArgsForCall(0)
-							Expect(err.Error()).To(ContainSubstring("parent type 'custom-type' error:"))
-						})
-
-						It("does not create a check", func() {
-							Expect(fakeCheckFactory.TryCreateCheckCallCount()).To(Equal(0))
+							err := fakeResourceType.SetCheckSetupErrorArgsForCall(0)
+							Expect(err.Error()).To(ContainSubstring("invalid duration not-a-duration"))
 						})
```

Comment:
This is an interesting scenario, if the parent type fail to check (i.e. bad config), should the child not be checked as well? 

This used to be the case before, but now that custom resource types are checked independently of their child resources, it's more complicated to string them together